### PR TITLE
add organisations schemes

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/organizations.json [hunk 1]
```diff
@@ -3,6 +3,105 @@
...
+            "CodeID": "code-03bfb698",
+            "CodeType": "Normal",
+            "DisplayText": "national government",
+            "NumericValue": 0,
+            "StringValue": "national_government",
+            "VisibleInCoda": true,
+            "MatchValues": [
+                "national government"
+              ]
+        },
```
Comment (as2388) [inline code_schemes/organizations.json]: Is this indentation the same as the one above?

File: code_schemes/organizations.json [hunk 2]
```diff
@@ -3,6 +3,105 @@
...
     "Codes": [
+      {
+            "CodeID": "code-03bfb698",
+            "CodeType": "Normal",
+            "DisplayText": "national government",
+            "NumericValue": 0,
+            "StringValue": "national_government",
+            "VisibleInCoda": true,
+            "MatchValues": [
+                "national government"
```
Comment (as2388) [inline code_schemes/organizations.json]: (what are these match values being used for?)

**Review by as2388 — APPROVED** [review]

File: code_schemes/organizations.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/organizations.json]: pipeline autocode

File: code_schemes/organizations.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/organizations.json]: Fixed indentation

**Pull request merged**